apps/tailwind-components/components/form/Legend.vue

```diff
@@ -14,7 +14,8 @@
         <span
```

Comment:
As mentioned above, the span should be a hyperlink. We could probably use a click event to scroll an element into view. Something like this:

```vue
<a href="..." @click="o'clock">...</a>
```

```js
function onClick(event: Event) => {
  event.prevent();
  const targetHref = document.querySelector(event.target as HTMLAnchorElement).getAttribute("href");
  document.querySelector(targetHref).scrollIntoView();
}
```

Reply:
We should also start supporting browser settings like reduce motion.

Reply:
as said will try. However because the elements first need resizing because of loading some magic might be needed still. Would be great if not.